Resolve #90

kijuky opened the pull request:
Resolve #90

kijuky committed:
Add sbt-test

kijuky committed:
Fix #90

xuwei-k commented on src/sbt-test/projectMatrix/lib/project/build.properties:
```diff
@@ -0,0 +1 @@
+sbt.version=1.9.9
```
don't add `build.properties` in scripted test

kijuky replied:
Thank you! Delete this file. :hand:

kijuky replied:
- Delete build.properties (use sbt 1.2.8)
- Changed the combination of Scala versions to be cross-compiled from (2.11, 3.3) to (2.12, 2.13)

kijuky committed:
Delete build.properties (use sbt 1.2.8)

- Delete build.properties (use sbt 1.2.8)
- Since Scala3 cannot be used with sbt1.2.8, change the combination of Scala versions to be cross-compiled from (2.11, 3.3) to (2.12, 2.13).

eed3si9n reviewed (approved):
Thanks!
